ui/src/tiles/Tile.tsx

```diff
@@ -66,6 +78,45 @@ export const Tile: FunctionComponent<TileProps> = ({
       onAuxClick={() => addRecentApp(desk)}
     >
       <div>
+        {desk === 'groups' && !hasOpenedGroups && (
+          <Popover.Root
```

Comment:
I think you want Tooltip here since it has the hover/focus functionality built in